src/fileTransformers/engineType.ts

```diff
@@ -0,0 +1,26 @@
+import { FileTransformer } from '../fileTransformer/fileTransformer'
+
+export const engineType: FileTransformer = (params) => {
+  const { file, parameters } = params
+
+  let content = file.content
+
+  if (file.path === 'prisma/schema.prisma') {
+    if (parameters.engineType) {
+      content = content.replace(
+        `provider = "prisma-client-js"`,
+        `provider = "prisma-client-js" \n engineType = "${parameters.engineType}"`
```

Comment:
Is this single space going to line up with the previous line? It might look weird. I think you'll need 2 spaces / a tab, whatever we use in the schema